roboteam_ai/include/roboteam_ai/utilities/GameState.h

```diff
@@ -6,6 +6,8 @@
 #include "Constants.h"
 #include "RuleSet.h"
 
+#include <roboteam_utils/Print.h>
```

Comment:
where is this used?

Reply:
I used quite some RTT_INFO and RTT_DEBUG statements to figure stuff out. Good catch. Will remove in next PR if it's not needed anymore